JS/models/ExperimentalTypes.js

```diff
@@ -1,8 +1,8 @@
-const ExperimentalTypes   = {
```

Comment:
Please use CAPITAL_LETTER notation for both constants and constant properties (if it is an object like it is in your case).